dizoo/evogym/envs/test/visualize_simple_env.py

```diff
@@ -0,0 +1,34 @@
+import gym
+from evogym import sample_robot
+from gym.wrappers import Monitor
+
+# import envs from the envs folder and register them
+import evogym.envs
+from dizoo.evogym.envs.viewer import DingEvoViewer
+from evogym.sim import EvoSim
+
+
+if __name__ == '__main__':
+    gym.logger.set_level(gym.logger.DEBUG)
+    # create a random robot
+    body, connections = sample_robot((5, 5))
+
+    # make the SimpleWalkingEnv using gym.make and with the robot information
+    #env = EvoGymEnv(EasyDict({'env_id': 'Walker-v0', 'robot': 'speed_bot', 'robot_dir': '../'}))
+    #env.enable_save_replay('video')
+
+    env = gym.make('Walker-v0', body=body)
+    env.default_viewer = DingEvoViewer(EvoSim(env.world))
+    env = Monitor(env, './video', force=True)
+    env.__class__.render = env.default_viewer.render
```

Comment:
Is there any better ways  to refine the code and place the render method inside the class?

Reply:
in diengine this process is not showed to the user, it is implemented in the wrapper class, no need to worry about this